.gitignore

```diff
@@ -13,4 +13,6 @@ UploadStatusFile*
 Tools/kvp_*
 Tools/jq
 Tools/gawk
-DownloadedFiles*
\ No newline at end of file
+Tools/sbinfo
```

Comment:
@juhlee-microsoft @harz566 Does any one upload this tool into our storage account? Thanks!

Reply:
@prash2611 Please email me the working binary. I will upload it